lib/install.ps1

```diff
@@ -476,7 +476,12 @@ function install_prog($fname, $dir, $installer) {
         if(!$installed) {
             abort "installation aborted. you might need to run 'scoop uninstall $app' before trying again."
         }
-        rm $prog
+        
+        #Don't remove installer if "keep" flag is set to true
+        if (!($keepInstaller.keep -eq "true"))
```

Comment:
You're completely correct; that was from a previous version of the changes while I was testing them. It should be fixed now; thanks for catching that!